components/Table/style.scss

```diff
@@ -46,44 +46,46 @@
   }
 }
 
-.markdown-body { 
+.markdown-body {
   @include markdown-table;
+
   .rdmd-table {
     $border-wrap-width: 1px;
+
     & {
-      position: relative;
       display: block;
+      position: relative;
     }
-    &:after {
-      content: ' ';
-      position: absolute;
-      box-shadow: inset 0 0 0 $border-wrap-width var(--table-edges, #dfe2e5);
-      width: 100%;
-      height: 100%;
-      top: 0;
-      pointer-events: none;
-      z-index: 1;
-    }
+
     &-inner {
-      min-width: 100%;
       box-sizing: content-box;
+      min-width: 100%;
+      overflow: auto;
       width: 100%;
     }
-    table:only-child {
-      margin: 0 !important;
```

Comment:
~I think we should keep this rule for consistency with other block elements in the engine.~

Reply:
lol jk you already got it

Reply:
Yeah the diff is just weird because I nested it in the new table rule 👍

Git Dif, Plz Git Gud?